src/main/java/org/snomed/aag/rest/AcceptanceController.java

```diff
@@ -61,10 +57,6 @@ public AcceptanceController(BranchSecurityService securityService,
 		// Update complete flag for all Criteria Items on this branch
 		Set<CriteriaItem> items = projectAcceptanceCriteriaService.findItemsAndMarkSignOff(projectAcceptanceCriteria, branchPath);
 
-		if (matchAuthorFlags) {
-			criteriaItemService.removeNonEnabled(items, branch);
-		}
-
```

Comment:
I've completely removed the "matchAuthorFlags" functionality so the ProjectAcceptanceCriteria is always in one state (as it is essentially formed from findEffectiveCriteriaWithMandatoryItems).